projects/storyteller/webapp/src/app/styles/feeds.scss

```diff
@@ -0,0 +1,64 @@
+$font-stack: OpenSans;
+%header {
```

Comment:
Why not `.header`? I'm not saying it's an error, I just didn't see that syntax before

Reply:
but if it's .header it's not working with inheritance in scss
https://github.com/hasadna/hasadna/pull/140/files/65f3fff4dbb7e3eae5f0c430ffd4f6d5dd8ffe46#diff-4dc0243ca547a4e54f5a6fd411ffd923R49

Reply:
What do you mean?